website/content/en/preview/concepts/disruption.md

```diff
@@ -78,15 +78,15 @@ Disruption is configured through the NodePool's disruption block by the `consoli
 ```yaml
 spec:
   disruption:
-    consolidationPolicy: WhenUnderutilized
+    consolidationPolicy: WhenEmptyOrUnderutilized
   template:
     spec:
       expireAfter: 720h
 ```
 {{% /alert %}}
 
 {{% alert title="Warning" color="warning" %}}
```

Comment:
I think this warning gets dropped now that we allow this consolidateAfter value to be set